README.md

```diff
@@ -94,72 +74,57 @@ export MODIN_ENGINE=dask  # Modin will use Dask
+engine based on [OmnisciDB](https://www.omnisci.com/platform/omniscidb) and included in the
+[Intel® Distribution of Modin](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/distribution-of-modin.html),
+which is a part of [Intel® oneAPI AI Analytics Toolkit (AI Kit)](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-analytics-toolkit.html).
 
-This gives you the flexibility to start with custom resource constraints and limit the
-amount of resources Modin uses.
 
+### Pandas API Coverage
 
-### Full Documentation
+<p align="center">
 
-Visit the complete documentation on readthedocs: https://modin.readthedocs.io
+| pandas Object     | Modin's Ray Engine Coverage                                                          | Modin's Dask Engine Coverage |
+|-------------------|:------------------------------------------------------------------------------------:|:---------------:|
+| `pd.DataFrame`    | <img src=https://img.shields.io/badge/api%20coverage-90.8%25-hunter.svg> | <img src=https://img.shields.io/badge/api%20coverage-90.8%25-hunter.svg> |
+| `pd.Series`       | <img src=https://img.shields.io/badge/api%20coverage-88.05%25-green.svg> | <img src=https://img.shields.io/badge/api%20coverage-88.05%25-green.svg> |
+| `pd.read_csv`     | ✅                                               | ✅ |
+| `pd.read_table`   | ✅                                               | ✅ |
+| `pd.read_parquet` | ✅                                               | ✅ |
+| `pd.read_sql`     | ✅                                               | ✅ |
+| `pd.read_feather` | ✅                                               | ✅ |
+| `pd.read_excel`   | ✅                                               | ✅ |
+| `pd.read_json`    | [✳️](https://github.com/modin-project/modin/issues/554)                                         | [✳️](https://github.com/modin-project/modin/issues/554) |
+| `pd.read_<other>` | [✴️](https://modin.readthedocs.io/en/latest/supported_apis/io_supported.html) | [✴️](https://modin.readthedocs.io/en/latest/supported_apis/io_supported.html) |
 
-### Scale your pandas workflow by changing a single line of code.
+</p>
+Some pandas APIs are easier to implement than others, so if something is missing feel
+free to open an issue!
 
+### More about Modin 
 
-```python
-import modin.pandas as pd
-import numpy as np
+Visit the complete documentation on readthedocs: https://modin.readthedocs.io/en/latest/index.html
 
-frame_data = np.random.randint(0, 100, size=(2**10, 2**8))
-df = pd.DataFrame(frame_data)
-```
-**In local (without a cluster) modin will create and manage a local (dask or ray) cluster for the execution**
+#### Scale your pandas workflow by changing a single line of code.
 
+**Note: In local mode (without a cluster), Modin will create and manage a local (Dask or Ray) cluster for the execution.**
```

Comment:
nit: the subheader and note are again a bit confusing to the eye/messy here.

Reply:
Do you think the Note's should be bolded? I think I can remove the bold?